drivers/serial/uart_native_tty_bottom.c

```diff
@@ -252,3 +348,42 @@ int native_tty_configure_bottom(int fd, struct native_tty_bottom_cfg *cfg)
 
 	return 0;
 }
+
+int native_tty_read_bottom_cfg(int fd, struct native_tty_bottom_cfg *cfg)
+{
+	struct termios ter;
+	int rc = 0;
+
+	rc = tcgetattr(fd, &ter);
+	if (rc != 0) {
+		int err = 0;
+
+		err = errno;
+		WARN("Could not read terminal driver settings: %s\n", strerror(err));
+		return -err;
```

Comment:
```suggestion
		return -nsi_errno_to_mid(err);
```
Note that as we (can) have different c libraries in the bottom and "top" side, the errno need to be translated.